doc/benchmark.conf.sample

```diff
@@ -37,6 +37,15 @@ lto = True
 # Profiled Guided Optimization (PGO)?
 pgo = True
 
+# The space-separated list of libraries that are package-only,
+# i.e., locally installed but not on header and library paths.
+# For each such library, determine the install path and add an
+# appropriate subpath to CFLAGS and LDFLAGS declarations passed
+# to configure. As an exception, the prefix for openssl, if that
+# library is present here, is passed via the --with-openssl
+# option. Currently, this only works with Homebrew on macOS.
+pkg_only = openssl zlib
```

Comment:
Should this be commented out in the default sample given that it will error out on non-Mac OS/X platforms?

Reply:
I considered that but decided to leave it in, since other options also have concrete values. But after going through the options again, I agree with you. Good catch!

Plus the actual list of libraries to make it all work on macOS is a tad longer, at least in the setup that worked for me. I'll run a few more experiments to hopefully find a minimal list and update the example, out-commenting it as well.